var/spack/repos/builtin/packages/py-sqlalchemy/package.py

```diff
@@ -12,6 +12,17 @@ class PySqlalchemy(PythonPackage):
     homepage = 'http://www.sqlalchemy.org/'
     pypi = "SQLAlchemy/SQLAlchemy-1.3.9.tar.gz"
 
+    # 'sqlalchemy.testing.suite' requires 'pytest'
```

Comment:
Please clarify the tie between the comment and the modules list added by this PR.

Reply:
The `import_modules` list added is the full list of detected modules minus `sqlalchemy.testing.suite`.

Reply:
Ah.  Perhaps change the comment to at least explain the module is not being included because of the dependency.

Reply:
I'm not sure if I've ever seen a single person other than myself use `import_modules`, so this comment is really only for my own posterity...

Reply:
@adamjstewart I think the bigger concern is if your comment isn't meaningful to other people, then you're effectively a bottleneck in development for anyone else who needs to understand the import modules here down the road -- I know you can keep a ton of context in your head, but someone else is going to need to touch this eventually and I think @tldahlgren is right that we need a more expressive comment for them. If I understand correctly, something like the following would work, but I'm actually still not even sure I understood the purpose of the comment.
```suggestion
    # 'sqlalchemy.testing.suite' requires 'pytest'
    # leave off import list to avoid dependency on  'pytest' that doesn't add functionality
```

Reply:
Done